dandiapi/urls.py

```diff
@@ -80,9 +81,11 @@ def to_url(self, value):
 register_converter(DandisetIDConverter, 'dandiset_id')
 urlpatterns = [
     path('', root_content_view),
+    path('robots.txt', robots_txt_view),
     path('api/', include(router.urls)),
     path('api/auth/token/', auth_token_view, name='auth-token'),
     path('api/stats/', stats_view),
+    path('api/stats/', stats_view),
```

Comment:
```suggestion
```

Reply:
whoops, forgot to remove -- should be removed now